### PR TITLE
Bratseth/lock tables

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

**Comment by freva** [comment]
27 test failures in node-repository

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java [hunk 1]
```diff
@@ -236,122 +199,76 @@ public void close() {
...
+        if (clusterTable.exists())
+            ensureClusterTableIsUpdated();
         else
-            createClusterTable(context);
+            createClusterTable();
+    }
+
+    private void ensureNodeTableIsUpdated() {
+        try {
+            if (0 == engine.getStatus(newContext().getCairoSecurityContext(), new Path(), nodeTable.name)) {
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java:216]: This is just `nodeTable.exists()`, but that is already checked before calling this the method?

**Review by freva — APPROVED** [review]

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java (hunk 1 above)
Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java:216]: Yes, it does nothing now, but I need it each time a column is added.

Comment (freva) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java:216]: Yeah, I get that, it's just that this `if` is redundant, could leave the `try` block empty until you need it

Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java]: Good point, done

**Pull request closed without merging**